### ISSUE TITLE
Arctic Fuse-like theme

### ISSUE THREAD

**Issue by puzavac** [ShadowBlip/OpenGamepadUI#402]
I see OGUI has few themes already though they mostly deal with color so I don't know how flexible OGUI is when it comes to themeing.

But if it is and if there are those who'd like to dabble with it, herby I propose an inspiration: Arctic Fuse theme for Kodi.

In fact I'd argue for more than inspiration(with permission of its author, of course); straight up Arctic Fuse for OGUI would be awesome. It's a beautiful theme and having both OGUI and Kodi with a unified look would make it a very attractive offering.

![image](https://github.com/user-attachments/assets/f55d0b6c-4a2a-49c1-b381-556e77133e6d)

![image](https://github.com/user-attachments/assets/6deda8aa-d1ca-4bcb-a3ff-b7af4e813643)

![image](https://github.com/user-attachments/assets/07c10eeb-7161-440a-9f86-111ed249553a)

**Comment by pastaq** [comment]
The color scheme and card shapes are very similar to the Dark Star theme we're already using. It probably wouldn't be difficult to adapt a few changes for a new theme for this.

**Comment by puzavac** [comment]
> The color scheme and card shapes are very similar to the Dark Star theme we're already using. It probably wouldn't be difficult to adapt a few changes for a new theme for this.

Yeah and that's great. Does OGUI allow for stronger modifications? Like, for example, PS5 has, what would be OGUI's quick menu, as some kind of dock at the bottom. Is such a change achievable in OGUI as part of more-or-less standard themeing? Or is quick menu position hardcoded?

**Comment by pastaq** [comment]
> > The color scheme and card shapes are very similar to the Dark Star theme we're already using. It probably wouldn't be difficult to adapt a few changes for a new theme for this.
> 
> Yeah and that's great. Does OGUI allow for stronger modifications? Like, for example, PS5 has, what would be OGUI's quick menu, as some kind of dock at the bottom. Is such a change achievable in OGUI as part of more-or-less standard themeing? Or is quick menu position hardcoded?

We can definitely do more themes, but there are essentially two types of themes that are possible:
- Color and object decoration
- Layout 

Changes to the location of the quick bar would fall under layout and require more time to code & create new UI objects. It's doable, but takes more work. Currently we only have color and object decoration themes.

I'm not familiar with the PlayStation UI at all, do you have a picture?

**Comment by puzavac** [comment]
> > > The color scheme and card shapes are very similar to the Dark Star theme we're already using. It probably wouldn't be difficult to adapt a few changes for a new theme for this.
> > 
> > 
> > Yeah and that's great. Does OGUI allow for stronger modifications? Like, for example, PS5 has, what would be OGUI's quick menu, as some kind of dock at the bottom. Is such a change achievable in OGUI as part of more-or-less standard themeing? Or is quick menu position hardcoded?
> 
> We can definitely do more themes, but there are essentially two types of themes that are possible:
> 
> * Color and object decoration
> * Layout
> 
> Changes to the location of the quick bar would fall under layout and require more time to code & create new UI objects. It's doable, but takes more work. Currently we only have color and object decoration themes.
> 
> I'm not familiar with the PlayStation UI at all, do you have a pic Here's an image. It's called Control Center in PS5 but from what I gathered it's basically your Quick Settings.

Is it doable with a lot of hacks or doable in the sense, yes, of course it takes more effort but steps were taken from the get go to make such efforts feel at home?

Here's an image
![image](https://github.com/user-attachments/assets/bf028a57-4a33-489c-911c-df2f72409006)


> > > The color scheme and card shapes are very similar to the Dark Star theme we're already using. It probably wouldn't be difficult to adapt a few changes for a new theme for this.
> > 
> > 
> > Yeah and that's great. Does OGUI allow for stronger modifications? Like, for example, PS5 has, what would be OGUI's quick menu, as some kind of dock at the bottom. Is such a change achievable in OGUI as part of more-or-less standard themeing? Or is quick menu position hardcoded?
> 
> We can definitely do more themes, but there are essentially two types of themes that are possible:
> 
> * Color and object decoration
> * Layout
> 
> Changes to the location of the quick bar would fall under layout and require more time to code & create new UI objects. It's doable, but takes more work. Currently we only have color and object decoration themes.
> 
> I'm not familiar with the PlayStation UI at all, do you have a picture?


Is it doable with a lot of hacks or doable in the sense, yes, of course it takes more effort but steps were taken from the get go to make such efforts feel at home?

Here's an image. It's called Control Center in PS5 but from what I gathered it's basically your Quick Settings:

![image](https://github.com/user-attachments/assets/bf028a57-4a33-489c-911c-df2f72409006)

**Comment by ShadowApex** [comment]
Yes, with some Godot knowledge, all parts of the UI can be modified or replaced completely via plugin. I think there's room for us to add more documentation and helper functions to make it easier, but with some know-how, you could accomplish this through the plugin system.

**Comment by this-is-just-con-tribute** [comment]
While we're on the subject of totally transforming the theme, something that I would love to see the community do is essentially create some sort of console-esque theme that is reminiscent of the sixth generation of consoles. You know, something that really makes you feel like you're inside the console, transporting you to a whole new world.

**Comment by puzavac** [comment]
> While we're on the subject of totally transforming the theme, something that I would love to see the community do is essentially creae a theme that is reminiscent of the sixth generation of consoles. You know, something that really makes you feel like you're inside the console, transporting you to a whole new world.

You'd like PS2-like theme? Fine by me if so, though I think PS5-like stuff is simply better overall. That's why I advocate for something like Arctic-Fuse.

**Comment by this-is-just-con-tribute** [comment]
Since it's possible to skin Kodi, wouldn't it make more sense to make an ogui skin to match it? I mean, your idea of having a theme that fits kodi makes sense, but why not skin kodi to match it? That being said, that theme looks sick! I hope it can happen somehow.

**Comment by puzavac** [comment]
Yes, that too can be. But OGUI theme, though not bad, isn't as sick. However perhaps someone implements PowerStation and InputPlumber in Kodi.
